TopicsImplementations/src/topiczerotwo/CommonBehavior.java

```diff
@@ -0,0 +1,32 @@
+package topiczerotwo;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * This class try to extract common behavior between connections.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public abstract class CommonBehavior implements SQLConnector {
```

Comment:
Keep in mind to use representative names, for the bootcamp is fine, but in a project you might use some class name like AbstractSQLConnector, or something that explains itself with its name